src/components/Search/SearchRouter/SearchRouterInput/index.native.tsx

```diff
@@ -125,6 +83,13 @@ function SearchRouterInput(
                         isLoading={!!isSearchingForReports}
                         ref={ref}
                         onKeyPress={handleKeyPress(onSubmit)}
+                        isMarkdownEnabled
+                        multiline={false}
+                        parser={(input: string) => {
+                            'worklet';
+
+                            return parseForLiveMarkdown(input, currentUserPersonalDetails.login ?? '', currentUserPersonalDetails.displayName ?? '');
+                        }}
```

Comment:
I have a general idea how to not repeat this whole component twice, and make it more obvious what changes are required for native VS web.

In short we could just have one component `SearchRouterInput` which would accept all needed params: selection, and parser.
And then we would have 2 wrapper components (we can come up with a name later) which would just be very small components, divided into `index` and `index.native` which would pass the required props.

But 95% of the logic and body of this Input would be in 1 file instead of two.
We can discuss this in more detail over slack.